src/Container/ApplicationFactory.php

```diff
@@ -91,7 +91,7 @@ public function __invoke(ContainerInterface $container)
      */
     private function injectRoutesAndPipeline(Application $app, array $config)
     {
-        $app->injectRoutesFromConfig($config);
-        $app->injectPipelineFromConfig($config);
+        ApplicationConfigInjectionDelegator::injectRoutesFromConfig($app, $config);
+        ApplicationConfigInjectionDelegator::injectPipelineFromConfig($app, $config);
```

Comment:
I did this to prevent triggering the deprecation notices in the trait when using this functionality.